pkg/vm/engine/tae/stl/types.go

```diff
@@ -30,9 +32,25 @@ func SizeOfMany[T any](cnt int) int {
 }
 
 type Bytes struct {
-	Data   []byte
-	Offset []uint32
-	Length []uint32
+	IsFixedType bool
```

Comment:
Why cant we just use the types type size stuff?   Where typeSize == -24 means varialble len, positive number means fixed type and fixed type size.   My point is -- do not invent new ways.

Reply:
It has been changed